app/assets/stylesheets/application_styles.scss

```diff
@@ -130,75 +130,115 @@ header.navbar {
 
 $photo-width: 187px;
 
```

Comment:
All of the changes in this file are aimed at making the items index page look the way it did before even though the markup is different.